ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/test-run-data-science-pipelines-operator-e2e-tests.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Documentation     Data Science Pipelines Operator E2E tests - https://github.com/opendatahub-io/data-science-pipelines-operator/tree/main/tests
+Suite Setup       Prepare Data Science Pipelines Operator E2E Test Suite
+Suite Teardown    Teardown Data Science Pipelines Operator E2E Test Suite
+Library           OperatingSystem
+Library           Process
+Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource          ../../../Resources/RHOSi.resource
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR}                /tmp/data-science-pipelines-operator
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_URL}           %{DATA-SCIENCE-PIPELINES-OPERATOR_REPO_URL=https://github.com/opendatahub-io/data-science-pipelines-operator.git}
+${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_REPO_BRANCH}        %{DATA-SCIENCE-PIPELINES-OPERATOR_REPO_BRANCH=main}
+${DSPANAMESPACE}                                          dspa-e2e
+${KUBECONFIGPATH}                                         %{HOME}/.kube/config
```

Comment:
@aloganat When running in Jenkins, is  `%{HOME}/.kube/config` correct for `${KUBECONFIGPATH}`?

Reply:
@jgarciao kconfig is available in the workspace directory on each pipeline.

Reply:
`%{HOME}/.kube/config` is working on Jenkins as well